hibernate-core/src/main/java/org/hibernate/dialect/CockroachDBDialect.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.dialect;
+
+import java.sql.Types;
+
+import org.hibernate.dialect.function.StandardSQLFunction;
+import org.hibernate.type.StandardBasicTypes;
+
+/**
+ * An SQL dialect for CockroachDB.
+ * Makes no changes at the moment, since CockroachDB now has the type aliases
+ * used. Keeping this file around for the future, though.
```

Comment:
"Now has the type aliases used" doesn't quite make sense to me. "Now provides type aliases for all types that Hibernate uses"?